src/content/test/semantics/headers-attribute.tsx

```diff
@@ -0,0 +1,118 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import { create, React } from '../../common';
+
+export const infoAndExamples = create(({ Markup, Link }) => (
+    <>
+        <h1>Headers attribute</h1>
+        <p>
+            The <Markup.Term>headers</Markup.Term> attribute of a <Markup.Term>{`<td>`}</Markup.Term> element must reference the correct{' '}
+            <Markup.Term>{`<th>`}</Markup.Term> element.
+        </p>
+
+        <h2>Why it matters</h2>
+        <p>
+            When assistive technologies describe data tables, they mention both the data cells and their programmatically-related header
+            cells. When incorrect header cells are reported, users are likely to find the table confusing. Proper use of the headers
```

Comment:
I think we should go back to Lisa Louise about this text. Either the word "proper" is wrong or the word "tedious" is wrong. Proper use would not make tables tedious. Either improper use would make tables tedious, or proper use would make tables easy to understand.

Reply:
Confirmed with Fer that we can remove the last sentence to unblock us.